EnglishFilter/sections/antiadblock.txt

```diff
@@ -606,40 +534,17 @@ hightech.web.id$$script[tag-content="adback_configuration"][min-length="100000"]
 @@||getfreebit.xyz/libs/advertisement.js
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39507
 @@||techperiod.com/wp-content/cache/min/*/wp-content/themes/TechPeriod/js/ads-*.js
-! https://github.com/AdguardTeam/AdguardFilters/issues/39505
-sendsurl.com#@#.ad-banner
-sendsurl.com#@##ad-banner
-sendsurl.com#%#AG_defineProperty('app_vars.force_disable_adblock', {value: "0"});
-! https://github.com/AdguardTeam/AdguardFilters/issues/39614
-geomica.com#@#.ad-banner
-geomica.com#@##ad-banner
-geomica.com#%#AG_defineProperty('app_vars.force_disable_adblock', {value: "0"});
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39501
 @@||turkdown.com/*/advertisement.js
 turkdown.com#$##agzblocker-unit-id { display: block!important; }
 turkdown.com#%#AG_setConstant('agzblocker.adsBlocked', 'noopFunc');
 turkdown.com#%#AG_abortInlineScript(/bannerad/, 'document.getElementById');
-! https://github.com/AdguardTeam/AdguardFilters/issues/39493
-@@||adshorted.net/ads.js
```

Comment:
Please re-add this rule:
`@@||adshorted.net/ads.js`

By the way, I think that hints `!+ NOT_PLATFORM(windows, mac, android)` are not necessary in this case.

Reply:
Re-added the rule and deleted platforms.